src/main.rs

```diff
@@ -10,11 +10,18 @@ use ztunnel::*;
 // #[global_allocator]
 // static GLOBAL: tcmalloc::TCMalloc = tcmalloc::TCMalloc;
 
-#[tokio::main(worker_threads = 2)]
-async fn main() -> anyhow::Result<()> {
+fn main() {
     telemetry::setup_logging();
     let config = ztunnel::config::Config {
         ..Default::default()
     };
-    app::spawn(signal::Shutdown::new(), config).await
+
+    tokio::runtime::Builder::new_multi_thread()
+        .worker_threads(config.num_worker_threads)
+        .enable_all()
+        .build()
+        .unwrap()
+        .block_on(async move {
+            let _ = app::spawn(signal::Shutdown::new(), config).await;
```

Comment:
This seems like we ignore errors now; previously tokio handled them. can we add it back?